code/game/machinery/computer/dna_console.dm

```diff
@@ -58,12 +104,26 @@
 		if(!isnull(connected))
 			break
 	injectorready = world.time + INJECTOR_TIMEOUT
+	scrambleready = world.time + SCRAMBLE_TIMEOUT
+	jokerready = world.time + JOKER_TIMEOUT
+
+	stored_research = SSresearch.science_tech
+
+/obj/machinery/computer/scan_consolenew/examine(mob/user)
+	..()
+	if(jokerready < world.time)
+		to_chat(user, "<span class='notice'>JOKER algorithm available.</span>")
+	else
+		to_chat(user, "<span class='notice'>JOKER algorithm available in about [round(0.00166666667 * (jokerready - world.time))] minutes.")
```

Comment:
Remember examine was refactored a while ago to use a list instead of multiple chat messages.